src/components/JoinUs/Learning.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { Padding, Margin } from 'styled-components-spacing'
+import styled from 'styled-components'
+import { Row, CompensatedRow, CompensatedCol } from '../grid'
+import Image from '../Common/Image'
+import { H2, Paragraph } from '../Typography'
+
+import {
+  Section,
+  FirstColumn,
+  SecondColum,
+  Separator,
+  List,
+  Hr,
+  TitleAndBody
+} from './elements'
+
+const InsightAnchor = styled.a`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  height: 100%;
+`
+
+const Learning = ({ data: { title, text, featuredInsights, list } }) => (
+  <Section greyBg>
+    <Row>
+      <FirstColumn>
+        <H2>{title}</H2>
+      </FirstColumn>
+      <SecondColum width={[1, 1, 1, 1, 6 / 12, 5 / 12]}>
+        <List list={list} />
+      </SecondColum>
+    </Row>
+    <Separator />
+    <TitleAndBody
+      title="Some insights into personal growth at YLD"
+      body={text}
+    />
+    <CompensatedRow>
+      {featuredInsights.map((el, idx) => (
+        <CompensatedCol width={[1, 1, 1, 4 / 12]} key={idx}>
+          <InsightAnchor href={el.url}>
+            <Padding top={2} style={{ width: '100%' }}>
+              <Margin bottom={1}>
+                <Image image={el.image} />
+              </Margin>
+            </Padding>
+            <Padding top={1} bottom={1} style={{ flex: 1 }}>
+              <Paragraph bold noMargin>
+                {el.image.title}
```

Comment:
Is it safe to assume that Greta will always ensure that the title of the image that she uploads starts with "YLD perspectives..."? i.e. that the title of the image that she uploads corresponds to the text we want to show on this page?

Perhaps we should use the "name" field in the content model for this:
![image](https://user-images.githubusercontent.com/15656538/51481498-e750c280-1d8b-11e9-8ff6-5467275333f4.png)

I.e. name = "Tom Gallacher" insetad of "Featured insight 1"
![image](https://user-images.githubusercontent.com/15656538/51481567-123b1680-1d8c-11e9-8a6c-892b05332924.png)

We would then hard code "YLD perspectives: " into this line of the codebase?

Reply:
> `I.e. name = "Tom Gallacher" instead of "Featured insight 1"`

I was not sure how you normally deal with this situations - I used `Featured insight 1` because it seemed reasonable — wasn't so sure on naming the model itself with the title of the link since that could change in the future. 
From content editor perspective, I'd prefer to be able to change the title to whatever I want (if for some reason I want to remove the 'YLD Perspectives' part, I can. I also thought that it was pretty clear that the current title is in the format `YLD Perspectives: name` so I thought it made sense.

I can change it to have it hardcoded, don't have much strong feelings regarding this

Reply:
Yeah that's fine about the hard coding. You're totally right about this:
> I'd prefer to be able to change the title to whatever I want (if for some reason I want to remove the 'YLD Perspectives' part, it's possible without a release).

I guess my main point was that it might be better to use a field in Contentful, that Greta explicitly has to type in, rather than grabbing the image's title?

Reply:
Makes sense, probably using a dedicated field is a best bet